src/ray/gcs/pubsub/gcs_pub_sub.cc

```diff
@@ -212,5 +213,87 @@ Status GcsSubscriber::SubscribeAllWorkerFailures(
   return Status::OK();
 }
 
+PythonGcsPublisher::PythonGcsPublisher(const std::string &gcs_address) {
+  std::vector<std::string> address = absl::StrSplit(gcs_address, ':');
+  RAY_LOG(DEBUG) << "Connect to gcs server via address: " << gcs_address;
+  RAY_CHECK(address.size() == 2);
+  gcs_address_ = address[0];
+  gcs_port_ = std::stoi(address[1]);
+}
+
+Status PythonGcsPublisher::Connect() {
+  grpc::ChannelArguments arguments;
+  arguments.SetInt(GRPC_ARG_MAX_MESSAGE_LENGTH, 512 * 1024 * 1024);
+  arguments.SetInt(GRPC_ARG_KEEPALIVE_TIME_MS, 60 * 1000);
+  arguments.SetInt(GRPC_ARG_KEEPALIVE_TIMEOUT_MS, 60 * 1000);
```

Comment:
Should we use the ray config value for these values? I found we have the same config for other Python client. 

```
Status PythonGcsClient::Connect() {
  grpc::ChannelArguments arguments;
  arguments.SetInt(GRPC_ARG_MAX_MESSAGE_LENGTH, 512 * 1024 * 1024);
  arguments.SetInt(GRPC_ARG_KEEPALIVE_TIME_MS, 60 * 1000);
  arguments.SetInt(GRPC_ARG_KEEPALIVE_TIMEOUT_MS, 60 * 1000);
  channel_ = rpc::BuildChannel(options_.gcs_address_, options_.gcs_port_, arguments);
  kv_stub_ = rpc::InternalKVGcsService::NewStub(channel_);
  runtime_env_stub_ = rpc::RuntimeEnvGcsService::NewStub(channel_);
  node_info_stub_ = rpc::NodeInfoGcsService::NewStub(channel_);
  job_info_stub_ = rpc::JobInfoGcsService::NewStub(channel_);
  return Status::OK();
```

Reply:
I dedup'ed the code now -- I don't think it is a great idea to introduce additional configs here (they were also hard coded in Python).

Once we are done with this migration, we can consider unifying the synchronous (wrapped by Python) and asynchronous (used by C++) configs :)